src/components/alert/Alert.js

```diff
@@ -0,0 +1,35 @@
+import icon from '../../assets/alert/alertIcon.svg';
+
+
+export default function Alert({ alert }) {
```

Comment:
Can we just add both type and text as props and not object? It is confusing that you have to create an object for this component.